### PR TITLE
Clarify use of use_column_value

### PR DESCRIPTION
**Pull request by karenzone**
Clarify use of use_column_value. 
Make last_run_metadata_path reference in record_last_run entry clickable.

Phase 1 fix for #273.
Phase 2 will involve reworking the topic

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -449,38 +449,39 @@ here, ":target_id" is a named parameter. You can configure named parameters
...
 
 [id="plugins-{type}s-{plugin}-tracking_column"]
-===== `tracking_column` 
+===== `tracking_column`
 
   * Value type is <<string,string>>
   * There is no default value for this setting.
 
-If tracking column value rather than timestamp, the column whose value is to be tracked
+The column whose value is to be tracked (if tracking column value rather than the last time the query was run)
```
Comment (jakelandis) [inline docs/index.asciidoc]: How about "The column whose value is to be tracked if use_column_value is set to true" ?

**Comment by jakelandis** [comment]
@karenzone - changes look good.  one minor suggestion. 

For phase 2, I will log a proposal issue. I think we can get rid of one (maybe two of these settings).

**Comment by jakelandis** [comment]
Logged https://github.com/logstash-plugins/logstash-input-jdbc/issues/279 for phase 2

**Comment by karenzone** [comment]
Yes, I tested without issue

**Pull request closed without merging**